libs/libc/signal/sig_delset.c

```diff
@@ -62,13 +62,12 @@
  *
  ****************************************************************************/
 
-int sigdelset(FAR sigset_t *set, int signo)
+int nxsig_delset(FAR sigset_t *set, int signo)
 {
   /* Verify the signal */
 
   if (!GOOD_SIGNO(signo))
     {
-      set_errno(EINVAL);
       return ERROR;
```

Comment:
```suggestion
      return -EINVAL;
```
There probably Return Value comments that need to be made to match too.